framework/PostProcessors/RavenOutput.py

```diff
@@ -1,259 +0,0 @@
-# Copyright 2017 Battelle Energy Alliance, LLC
```

Comment:
Removing this file is not mentioned in the description of this merge.  Can you edit the pull request description to explain?  Thanks.  (Otherwise I am fine with this pull.)

Reply:
done